content/riak/kv/2.0.0/developing/usage/conflict-resolution.md

```diff
@@ -541,11 +541,13 @@ obj.store
 bucket = client.bucket_type('siblings_allowed').bucket('nickolodeon')
 obj = bucket.get('best_character')
 
-# Then we modify the object's value
-new_obj.data = 'Stimpy'
+# Then pick one of the siblings
+resolved_sibling = obj.siblings[3]
 
-# Then we store the object, which has the vector clock already attached
-new_obj.store(vclock=vclock)
+# Then replace the siblings data structure with the single sibling we want to keep
+obj.siblings = [resolved_sibling]
+obj.store()
+obj.reload()
```

Comment:
Is reload necessary? I see that `reload()` is called in the tests but as far as I know `store()` should suffice. It could be tested by removing the call to `reload()` [here](https://github.com/basho/riak-python-client/blob/master/riak/tests/test_kv.py#L438) and running the tests.